guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -83,4 +83,7 @@ Where `_mailca.crt_` is the CA certificate of the SMTP server.
 If a message fails to send, the {ProjectWebUI} displays an error.
 See the log at `/var/log/foreman/production.log` for further details.
 
-NOTE: For information on configuring email notifications for individual users or user groups, see {AdministeringDocURL}Configuring_Email_Notification_Preferences_admin[Configuring Email Notification Preferences] in _{AdministeringDocTitle}_.
+[NOTE]
```

Comment:
You could convert this to ".Additional resources" and an unordered list.

Reply:
Agree. It's a pain to use 'searches' to track down assemblies. I wish the repo followed the convention of adding comments to modules: "// This module is used in the following assemblies:"

Reply:
Now I see what you mean. Foreman has "Additional resource" sections in modules. That's new to me!